tasks/install_tasks.py

```diff
@@ -48,6 +50,26 @@ def install_tools(ctx):
                 with ctx.cd(path):
                     for tool in tools:
                         ctx.run(f"go install {tool}")
+        install_custom_golanci_lint(ctx)
+
+
+def install_custom_golanci_lint(ctx):
+    res = ctx.run("golangci-lint custom -v")
+    if res.ok:
+        gopath = os.getenv('GOPATH')
+        gobin = os.getenv('GOBIN')
+
+        if gopath is None and gobin is None:
+            print("Not able to install custom golangci-lint binary. golangci-lint won't work as expected")
+            raise Exit(code=1)
+
+        if gobin is not None and gopath is None:
+            shutil.move("golangci-lint", f"#{gobin}/golangci-lint")
+
+        if gopath is not None:
+            shutil.move("golangci-lint", f"{gopath}/bin/golangci-lint")
```

Comment:
Can we backup the previous versions of the binaries so people can easily switch back to them if needed?

Reply:
Of we can 🎉 

We can rename to `original-golangci-lint`

What do you think? @FlorentClarret